src/build.props

```diff
@@ -9,8 +9,9 @@
     <Company Condition=" '$(Company)' == '' ">Microsoft</Company>
     <Product Condition=" '$(Product)' == '' ">Microsoft SARIF SDK</Product>
     <Copyright Condition=" '$(Copyright)' == '' ">© Microsoft Corporation. All rights reserved.</Copyright>
-    <VersionPrefix Condition=" '$(VersionPrefix)' == ''">2.0.0</VersionPrefix>
-    <VersionSuffix Condition=" '$(VersionSuffix)' == ''">csd.2.beta.2019-01-24</VersionSuffix>
+    <VersionPrefix>2.0.0</VersionPrefix>
+    <VersionSuffix>csd.2.beta.2019-01-24</VersionSuffix>
+    <PackageVersionSuffix>.1</PackageVersionSuffix>
```

Comment:
>PackageVersionSuffix [](start = 5, length = 20)

Now that I'm done with this change, it's clear I did it the wrong way. Version suffix should be constructed from a core suffix + optional package patch, like .1. This allow VersionSuffix to naturally flow to the dotnet pack command. The powershell to create the version constants code should use the constituent elements comprising VersionSuffix to create that code. I can make this update later.

Reply:
Ok, I'll wait to see your update.

---
In reply to: [259148357](https://github.com/Microsoft/sarif-sdk/pull/1304#discussion_r259148357) [](ancestors = 259148357)

Reply:
Oh, your other comment says you're not going to update, so I'll continue this review.

---
In reply to: [259148635](https://github.com/Microsoft/sarif-sdk/pull/1304#discussion_r259148635) [](ancestors = 259148635,259148357)